crates/oxc_minifier/src/ast_passes/fold_constants.rs

```diff
@@ -54,23 +50,27 @@ impl<'a> FoldConstants<'a> {
         self
     }
 
-    fn fold_expression_and_get_boolean_value(&mut self, expr: &mut Expression<'a>) -> Option<bool> {
-        self.fold_expression(expr);
-        get_boolean_value(expr)
+    fn fold_expression_and_get_boolean_value(
```

Comment:
It feels like all these fold methods should go into a trait as well, I'll try in another PR tomorrow.